async_upnp_client/server.py

```diff
@@ -103,6 +109,118 @@ class NopRequester(UpnpRequester):  # pylint: disable=too-few-public-methods
     """NopRequester, does nothing."""
 
 
+class EventSubscriber:
+    """Represent a service subscriber."""
+
+    DEFAULT_TIMEOUT = 3600
+
+    def __init__(self, callback_url: str, timeout: Optional[int]) -> None:
+        """Initialize."""
+        self._url = callback_url
+        self._uuid = str(uuid4())
+        self._event_key = 0
+        self._expires = datetime.now()
+        self.timeout = timeout
```

Comment:
Should this be `self.timeout = timeout or self.DEFAULT_TIMEOUT`? I'm not sure, `self.DEFAULT_TIMEOUT` is used in the `self.timeout` setter.

Reply:
I don't understand this comment.  If the user passes `None` as the argument, then the setter will set the value to DEFAULT_TIMEOUT.  if the user passes a number, it will be preserved.

Reply:
Ah, I was mislead by all previous lines setting the instance variables, but this one using a setter.